File: openapi/code_samples/java/api_company_bankrupt_activeYears.java
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class CompanyController {

    @GetMapping("/api/company/bankrupt-companies-active-years")
    public ResponseEntity<?> getBankruptCompaniesActiveYears(
            @RequestParam(name = "activityYears", defaultValue = "3") int activityYears,
            @RequestParam(name = "caen", defaultValue = "") String caen,
            @RequestParam(name = "cui", defaultValue = "") String cui,
            @RequestParam(name = "county", defaultValue = "") String county,
            @RequestParam(name = "pageNumber", defaultValue = "0") int pageNumber,
            @RequestParam(name = "pageSize", defaultValue = "20") int pageSize,
            @RequestParam(name = "sortBy", defaultValue = "cui") String sortBy) {

        // Your implementation here

        return ResponseEntity.ok().build();
    }
}

File: openapi/code_samples/python/api_company_bankrupt_activeYears.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/bankrupt-companies-active-years', methods=['GET'])
def get_bankrupt_companies_active_years():
    activity_years = int(request.args.get('activityYears', 3))
    caen = request.args.get('caen', '')
    cui = request.args.get('cui', '')
    county = request.args.get('county', '')
    page_number = int(request.args.get('pageNumber', 0))
    page_size = int(request.args.get('pageSize', 20))
    sort_by = request.args.get('sortBy', 'cui')

    # Your implementation here

    return jsonify({}), 200

if __name__ == '__main__':
    app.run()
